source/templates.tex

```diff
@@ -2216,11 +2216,11 @@
 \begin{example}
 A single class template
 \tcode{List}
-might provide an unbounded set of class definitions:
+provides an unbounded set of class definitions:
 one class \tcode{List<T>} for every type \tcode{T},
 each describing a linked list of elements of type \tcode{T}.
 Similarly, a class template \tcode{Array} describing a contiguous,
-dynamic array might be defined like this:
+dynamic array can be defined like this:
 \begin{codeblock}
 template<class T> class Array {
   T* v;
```

Comment:
(Can't comment on this directly because github is terrible.) There's a bogus "may" a few lines below that should be a "can". ISO/CS noticed and called out this one in our discussion so we need to fix it.

Reply:
may -> can